Formula/freeswitch.rb

```diff
@@ -186,12 +155,15 @@ def install
 
     system "./bootstrap.sh", "-j"
 
-    system "./configure", "--disable-dependency-tracking",
-                          "--enable-shared",
-                          "--enable-static",
-                          "--prefix=#{prefix}",
-                          "--exec_prefix=#{prefix}"
+    args = std_configure_args + %W[
+      --enable-shared
+      --enable-static
+      --exec_prefix=#{prefix}
+    ]
+    # Fails on ARM: https://github.com/signalwire/freeswitch/issues/1450
+    args << "--disable-libvpx"
```

Comment:
```suggestion
    args << "--disable-libvpx" if Hardware::CPU.arm?
```
?

Reply:
Wasn't sure if having inconsistent codec support between architectures would be a problem, so just disabled it for all.

Reply:
We usually just disable things on ARM if they don't work specifically on ARM though.